certification/internal/policy/container/runnable_container.go

```diff
@@ -0,0 +1,66 @@
+package container
+
+import (
+	"fmt"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/internal/cli"
+	log "github.com/sirupsen/logrus"
+)
+
+// RunnableContainerCheck starts a container and
+// ensures it reaches a Running state within a specified timeframe
+type RunnableContainerCheck struct {
+	PodmanEngine cli.PodmanEngine
+}
+
+func NewRunnableContainerCheck(podmanEngine *cli.PodmanEngine) *RunnableContainerCheck {
+	return &RunnableContainerCheck{
+		PodmanEngine: *podmanEngine,
+	}
+}
+
+func (p *RunnableContainerCheck) Validate(imgRef certification.ImageReference) (bool, error) {
+	inspectionData, err := p.PodmanEngine.InspectImage(imgRef.ImageURI)
+	if err != nil {
+		log.Error(fmt.Sprintf("unable to inspect the image %s: ", imgRef.ImageURI), err)
+		return false, nil
+	}
+
+	cmd := []string{inspectionData.Config.Entrypoint}
+	cmd = append(cmd, inspectionData.Config.Cmd...)
+	cmd = append(cmd, "&&", "sleep", "60")
```

Comment:
I could see this going either way:
- we append the sleep to the existing cmd inside the container, which is what is being done above.
- we overwrite the existing cmd inside the container with _just_ a `sleep 60`.

I'm not sure which is the correct approach for what we are trying to accomplish.

Reply:
The third approach could be to read the command from viper. @bcrochet @jomkz @tonytcampbell: I appreciate your thoughts/feedbacks.

Reply:
I don't think getting from viper is the right choice. It seems to me that we want a consistent result, so I'd opt for option 2. Just calling sleep 1. Do we need to actually sleep 60?